src/js/components/DataTable/buildState.js

```diff
@@ -84,12 +84,34 @@ export const filterAndSortData = (data, filters, onSearch, sort) => {
   if (sort && !sort.external) {
     const { property, direction } = sort;
     result = result === data ? [...data] : result; // don't sort caller's data
-    const before = direction === 'asc' ? 1 : -1;
-    const after = direction === 'asc' ? -1 : 1;
+    const sortAsc = direction === 'asc';
+    const before = sortAsc ? 1 : -1;
+    const after = sortAsc ? -1 : 1;
     result.sort((d1, d2) => {
-      if (datumValue(d1, property) > datumValue(d2, property)) return before;
-      if (datumValue(d1, property) < datumValue(d2, property)) return after;
-      return 0;
+      const d1Val = datumValue(d1, property);
+      const d2Val = datumValue(d2, property);
+      const d1IsString = typeof d1Val === 'string';
+      const d2IsString = typeof d2Val === 'string';
+      const d1ValProc = d1IsString ? d1Val.toLowerCase() : d1Val;
```

Comment:
why do you do toLowerCase when you're already going to use { sensitivity: 'base' } ?

Reply:
D'oh! Good call. Fixed.